tests/js-authentication/authentication.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We've already got a `jwt_test.py` test. Perhaps add it there instead?

Reply:
The new test is jwt only, yes, but the intended purpose is to cover other auth schemes as well, looking at #1904. Since we have folders now for each JS test app it makes sense to have the test module in that folder as well, like a self-contained package. Another argument is that jwt_test.py is for governance things, where the new one is for app/runtime logic.

Reply:
OK, makes sense. Does the `js-authentication` JS app currently double verify the token (once natively, once in JS)?

Reply:
No, just natively.